fmriprep/run_workflow.py

```diff
@@ -97,7 +96,7 @@ def main():
     opts = parser.parse_args()
 
     if opts.version:
-        print('fmriprep version ' + __version__)
+        logging.info('fmriprep version ' + __version__)
```

Comment:
Why should we use logging for an option that only shows the current version? Isn't it a little bit of overkilling the problem?